project1/import.py

```diff
@@ -0,0 +1,41 @@
+import os
+import csv
+from sqlalchemy import create_engine
+from sqlalchemy.orm import scoped_session, sessionmaker
+from flask import Flask
+from flask_session import Session
+from flask_sqlalchemy import SQLAlchemy
+
+
+app1 = Flask(__name__)
+
+app1.config["SQLALCHEMY_DATABASE_URI"] = os.getenv("DATABASE_URL")
+app1.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
+app1.app_context().push()
+db1 = SQLAlchemy()
+
+class Books(db1.Model):
+    __tablename__ = "BOOKS"
+    isbn = db1.Column(db1.String, nullable = False,primary_key=True)
+    tittle = db1.Column(db1.String, nullable = False)
+    author = db1.Column(db1.String, nullable = False)
+    year = db1.Column(db1.String, nullable= False)
+
+db1.init_app(app1)
+db1.create_all()
+
+def main():
+    f= open("books.csv")
+    reader = csv.reader(f)
+    for isbn, tittle, author, year in reader:
+        book = Books(isbn= isbn, tittle=tittle, author=author, year=year)
+        db1.session.add(book)
+    db1.session.commit()
```

Comment:
Are you able to load all books records into your table?

Reply:
yes mam i am able to 
load all the books ..it showed me 5001 rows after everything is done

Reply:
:+1: